war/src/main/less/base/layout-commons.less

```diff
@@ -125,6 +125,8 @@ body.full-screen #main-panel {
 /* -------------------------------------- */
 
 h1.build-caption.page-headline {
+    display: flex;
```

Comment:
Either this or the build status icon change has caused the progress bar to move to the left during builds in the console log and on the run page

![image](https://user-images.githubusercontent.com/21194782/150493910-f58a25b3-c6ea-488a-aea3-4f11b5441cbb.png)